README.md

```diff
@@ -327,6 +327,36 @@ admin/configure add publishing-all-ports
 sudo docker-compose up -d
 ```
 
+#### Modify memory settings
```

Comment:
Great! How about adding "NOTE: You should consider modifying your memory settings in order to give your database a sufficient amount of ram, otherwise your database could run very slowly. See section Modify memory settings." to the #create-database?

Reply:
Should be good now. I did the same for Solr. See https://github.com/metabrainz/musicbrainz-docker/pull/148/commits/273c65ee2d16106baa8ef1a28ddaed02008ea812, https://github.com/metabrainz/musicbrainz-docker/pull/148/commits/b298cbe3a42b3d1a604c554e119fd37097cc693e, and https://github.com/metabrainz/musicbrainz-docker/pull/148/commits/8821ad4c2c2db3a53b40de3bb76cb9cad63cebef.